CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -20,6 +20,9 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Compile binaries using go [1.21.11](https://github.com/etcd-io/etcd/pull/18130).
 - Upgrade [bbolt to 1.3.10](https://github.com/etcd-io/etcd/pull/17945).
 
+### Package clientv3
+- [Print gRPC metadata in guarnteed order using the official go fmt pkg](https://github.com/etcd-io/etcd/pull/18311).
+
```

Comment:
3.4.33 has already released. Please put this under 3.4.34 (above the section `Dependencies`)

Reply:
Thanks! updated

Reply:
This was my mistake mentioning 3.4.33 in my feedback above instead of 3.4.34, sorry for the confusion.

Reply:
np at all :)